### PR TITLE
fix(defaults): Remove the tasklet from around the config update

### LINKED ISSUE #125: config updated message from all commands
SUCCESS  Configfile updated (0s)
is printed as the first output from every command, it probably shouldn't be updated for all commands or at least not printed.

### PR DESCRIPTION
**Pull request by asalkeld**
The update message is not needed.

fixes #125

### CODE REVIEW

**Review by davemooreuws — APPROVED** [review]

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -153,18 +152,12 @@ func ensureConfigDefaults() {
 	}
 
 	if needsWrite {
```
Comment (tjholm) [inline pkg/cmd/root.go:154]: is needsWrite always true? It seems to always run

Comment (asalkeld) [reply pkg/cmd/root.go:154]: I know mine definitely doesn't..

Comment (tjholm) [reply pkg/cmd/root.go:154]: Running nitric version on a build from develop
image

Comment (tjholm) [reply pkg/cmd/root.go:154]: Running version on the latest dev build:
image

I know removing the tasklet will remove the output, but it does run every time on my machine.

**Comment by asalkeld** [comment]
not relevant any more

**Pull request closed without merging**